trikot-viewmodels-declarative-flow/sample/ios/Sample/SampleSpanStyleProvider.swift

```diff
@@ -0,0 +1,15 @@
+import SwiftUI
+import TRIKOT_FRAMEWORK_NAME
+import Trikot
+
+class SampleSpanStyleProvider: VMDSpanStyleProvider {
```

Comment:
Should we change it to SampleTextStyleProvider since we changed to VMDTextStyleResource. Same for VMDSPanStyleProvider?

Reply:
I hesitated too. My reasoning is that it's only used in a rich text context. The type provides "span styles" for attributed strings formatting, and does so based on a `VMDTextStyleResource` parameter.